backend/Shared/Logger.js

```
@@ -0,0 +1,81 @@
+      var result = {};
+      if (mod != null)
+        for (var k = ownKeys(mod), i = 0; i < k.length; i++)
+          if (k[i] !== 'default') __createBinding(result, mod, k[i]);
+      __setModuleDefault(result, mod);
+      return result;
+    };
+  })();
+Object.defineProperty(exports, '__esModule', { value: true });
+exports.httpLogger = void 0;
+exports.addApiBreadcrumbs = addApiBreadcrumbs;
+const crypto_1 = require('crypto');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)